event_routing_backends/helpers.py

```diff
@@ -23,6 +23,23 @@
 UTC_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%f'
 
 
+def get_uuid5(namespace_key, name):
+    """
+    Create a UUID5 string based on custom namesapce and name.
+
+    Arguments:
+    namespace_key (str):    key to be used to create a custom namespace
+    name  (str):            string for which we need to creat a UUID
+
+    Returns:
+        str
+
+    """
+    base_uuid = uuid.UUID('6ba7b810-9dad-11d1-80b4-00c04fd430c8')
```

Comment:
Should the `base_uuid` be a setting?

Reply:
@Ian2012 I don't think it matters much, it's just for namespacing purposes not cryptography or anything. I'd be much more concerned about someone changing that setting and accidentally causing catastrophic data duplication

Reply:
can we add a comment specifying why it is hardcoded?